features/user_can_navigate_site.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Navbar
+  As a user
+  In order to navigate the site
+  I would like to see a navbar with a home button
+
+  Scenario: Display navbar and allow visitor to navigate back home
+    Given I visit the landing page
+    Then the "navigation-bar" should be visible
```

Comment:
Feels a bit too technical IMO to be part of the scenario... Can't you rephrase this in the scenario and use the class name in step definition?

Reply:
👍